Source/WebCore/html/HTMLIFrameElement.h

```diff
@@ -83,6 +88,9 @@ class HTMLIFrameElement final : public HTMLFrameElementBase {
     bool m_IFrameFullscreenFlag { false };
 #endif
     std::unique_ptr<LazyLoadFrameObserver> m_lazyLoadFrameObserver;
+#if ENABLE(CONTENT_EXTENSIONS)
+    URL m_initiatorSourceURL;
```

Comment:
Why do we need to store this? Why can we check `document().url()` ?

Reply:
`document().url()` is the frame's URL, but we want to know who create `iframe`. Even if the frame's url is not eligible for checking, it is possible the script is eligible and to owner that decision, we need this.